docs/kqueen.rst

```diff
@@ -151,18 +151,155 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
     * - PROMETHEUS_WHITELIST
       - 127.0.0.0/8
       - Addresses allowed to access metrics endpoint without token
+Before you provision a Kubernetes cluster, you may need to deploy and configure the following external services:
+* Mail server, to enable members management. For example, to enable a user to invite other users by email. You can use the KQueen predefined mail service or run a new one.
+* Prometheus server, to extend the monitoring in KQueen. You can either use a predefined Prometheus server, defined in the docker-compose.production.yml file or use an external one. For an external one, you must include the rules from kqueen/prod/prometheus to the existing Prometheus service.
 
-
-Backup
+**To set up the mail server**
 -------
```

Comment:
Remove ** from header and set ------- length equal to the text length so all formatting will be done automatically.
 And for all headers below do the same, please.